ibis/backends/dask/__init__.py

```diff
@@ -62,3 +62,5 @@ def register_options(self):
             'See ibis.dask.trace for details.',
             validator=ibis.config.is_bool,
         )
+        # forces the pandas backend to register options
+        getattr(ibis, "pandas")
```

Comment:
you can just `import ibis.backends.pandas` right? (in the dask implementation itself)

Reply:
but agree we should handle the options registration in a better way

Reply:
There's a tracking issue here: https://github.com/ibis-project/ibis/issues/2844. This was mostly to get my dask tests working locally. Importing would accomplish the same thing, but generally should be fixed at the ibis level.